src/views/index.ejs

```diff
@@ -33,15 +33,29 @@
 
     <script src="<%- launchEmbedUrl %>" async></script>
 
-<% if (isProduction) { %>
-    <!-- Minified Styles -->
-    <link rel="stylesheet" type="text/css" href="<%- baseUrl %>/styles.css">
-<% } %>
+    <% if (isProduction) { %>
+    	<!-- Minified Styles -->
+    	<link rel="stylesheet" type="text/css" href="<%- baseUrl %>/styles.css">
+    <% } %>
+
+    <!--min height on header container to avoid jump on page load-->
+    <style>
+      #Header-Placeholder {
+        min-height: 70px;
+      }
+      @media screen and (min-width: 1024px) {
```

Comment:
Two minor updates: the media query should be 832px and the min height should be 130px instead of 230px.